website/source/docs/secrets/rabbitmq/index.html.md

```diff
@@ -0,0 +1,326 @@
+---
+layout: "docs"
+page_title: "Secret Backend: RabbitMQ"
+sidebar_current: "docs-secrets-rabbitmq"
```

Comment:
There should be an entry in `website/source/layouts/docs.erb` for `docs-secrets-rabbitmq` for this to work.